includes/event/event-date.php

```diff
@@ -58,6 +58,15 @@ public function is_in_the_past() {
 		return $this->utc() < $current_date_time;
 	}
 
+	/**
+	 * Get the date in seconds.
+	 *
+	 * @return int The date in seconds.
+	 */
+	public function __toSeconds(): int {
+		return $this->utc()->getTimestamp();
+	}
+
```

Comment:
I think we are not using this method. Can we remove it, or are you planning to use it in another place?

```suggestion
```

Reply:
Thanks good catch!